docs(node-v): fixes node version in documentation (#2375)
UMAprotocol/protocol: README.md

### Install dependencies 👷‍♂️

> You'll need to install a 12.x.x version of nodejs, since later versions will break the repo. You will also need to install yarn. Assuming that's done, run `yarn` with no args:

Reviewer: ```suggestion
You'll need to install a 12.x.x version of nodejs, since other versions will break some dependencies. You will also need to install yarn. Assuming that's done, run `yarn` with no args:
```
